aws-rds-dbcluster/src/main/java/software/amazon/rds/dbcluster/Translator.java

```diff
@@ -189,9 +189,9 @@ static DeleteDbClusterRequest deleteDbClusterRequest(
         return builder.build();
     }
 
-    static RemoveFromGlobalClusterRequest removeFromGlobalClusterRequest(final ResourceModel model) {
+    static RemoveFromGlobalClusterRequest removeFromGlobalClusterRequest(final ResourceModel model, final String clusterArn) {
         return RemoveFromGlobalClusterRequest.builder()
-                .dbClusterIdentifier(model.getDBClusterIdentifier())
+                .dbClusterIdentifier(clusterArn)
                 .globalClusterIdentifier(model.getGlobalClusterIdentifier())
                 .build();
```

Comment:
looks good to me, did you test the change?

Reply:
Yes, I submitted resource to my AWS account and executed this stack against
```
{
  "Description": "Basic DBCluster Template",
  "Parameters": {
    "DBClusterId": {
      "Type": "String"
    },
    "GlobalClusterId": {
      "Type": "String"
    }
  },
  "Resources": {
    "BasicDBCluster": {
      "Type": "AWS::RDS::DBCluster",
      "DeletionPolicy" : "Delete",
      "Properties": {
        "Engine": "aurora",
        "MasterUsername": "guest",
        "MasterUserPassword": "guest123",
        "EngineVersion" : "5.6.mysql_aurora.1.22.2",
        "DBClusterIdentifier": { "Ref": "DBClusterId" },
        "GlobalClusterIdentifier": { "Ref": "GlobalClusterId" }
      }
    }
  }
}
```